aio/deploy/helm-chart/kubernetes-dashboard/README.md

```diff
@@ -102,7 +102,8 @@ Parameter                                       | Description
 `ingress.customPaths`                           | Override the default ingress paths                                                                                               | `[]`
 `ingress.hosts`                                 | Dashboard Hostnames                                                                                                              | `nil`
 `ingress.tls`                                   | Ingress TLS configuration                                                                                                        | `[]`
-`settings`                                      | Global dashboard settings | `{}`
+`settings`                                      | Global dashboard settings                                                                                                        | `{}`
+`pinnedCRDs`                                    | Pinned CRDs in dashboard                                                                                                         | `{}`
```

Comment:
Could you add that they are pinned as the menu entries?

Reply:
Fixed :+1: